### ISSUE TITLE
Updated existing POCs to CDEvents v0.1

### ISSUE THREAD

**Issue by afrittoli** [cdevents/community#18]
CDEvents has two POCs in https://github.com/cdfoundation/sig-events/tree/main/poc.
Those two POCs:
- are hosted in the sig-events repo
- use an old version of the golang SDK
- have somewhat sparse documentation
- are not discoverable through cdevents.dev

This issue is about fixing all this. It would be ok to split it in sub-issues if needed.

**Comment by e-backmark-ericsson** [comment]
See also the small related comment here: https://github.com/cdfoundation/sig-events/issues/128#issuecomment-1223918341

**Comment by afrittoli** [comment]
@rjalander is this something you intend to work on / contribute to?

**Comment by rjalander** [comment]
Yes @afrittoli, we have in our backlogs to work on this Updating PoC version to use CDEvents v0.1

**Comment by afrittoli** [comment]
@andrew-est would you be able to share your recent POC? That way we could replace the old one.
